godtossing/Pos/Ssc/GodTossing/SecretStorage.hs

```diff
@@ -6,12 +6,12 @@ module Pos.Ssc.GodTossing.SecretStorage
 
 import           Universum
 
-import           Pos.Binary.Ssc           ()
-import           Pos.DB                   (MonadDB)
-import           Pos.DB.Misc.Common       (miscGetBi, miscPutBi)
-import           Pos.Ssc.GodTossing.Core  (Opening, SignedCommitment)
-import           Pos.Ssc.GodTossing.Types (GtSecretStorage (..))
-import           Pos.Types                (EpochIndex)
+import           Pos.Binary.Ssc.GodTossing.Types ()
```

Comment:
Perhaps we can get rid of `Ssc` in `Pos.Binary.Ssc.GodTossing`, especially now when `GodTossing` is in a separate package.